lib/pbench/cli/server/shell.py

```diff
@@ -25,19 +29,71 @@ def app():
     return create_app(server_config)
 
 
+def add_to_path(path_el: str, path: str) -> str:
+    """Convenience function to hide f-string syntax used to construct a new
+    PATH environment variable.
+
+    Arguments:
+
+        path_el - a new path element string to be pre-pended to `path`
+        path - the exiting PATH environment variable value
+
+    Returns:
+
+        A string representing the newly constructed PATH value
+    """
+    return f"{path_el}:{path}"
+
+
 def find_the_unicorn(logger: Logger):
-    if site.ENABLE_USER_SITE:
-        local = Path(site.getuserbase()) / "bin"
-        if (local / "gunicorn").exists():
-            # Use a `pip install --user` version of gunicorn
-            os.environ["PATH"] = str(local) + ":" + os.environ["PATH"]
-            logger.info(
-                "Found a local unicorn: augmenting server PATH to {}",
-                os.environ["PATH"],
-            )
+    """Add the location of the `pip install --user` version of gunicorn to the
+    PATH if it exists.
+    """
+    local = Path(site.getuserbase()) / "bin"
+    if (local / "gunicorn").exists():
+        # Use a `pip install --user` version of gunicorn
+        os.environ["PATH"] = add_to_path(local, os.environ["PATH"])
```

Comment:
This change seems like it creates more obfuscation than clarity or even convenience.

`add_to_path` isn't a sufficiently descriptive name, since it doesn't indicate whether the new path component is added to head or the tail of the path...whereas this detail was obvious looking at the old code.

Moreover, the operation isn't so complicated or specialized that we need to avoid duplicating it (and, the replacement seems to take up just as much space as the original at the call site).

If you really don't like using `+` for string concatenation, we could go with something like
```
        os.environ["PATH"] = ":".join((local, os.environ["PATH"]))
```
😉